mne/viz/epochs.py

```diff
@@ -169,14 +169,14 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
     if combine is not None:
         ts_args["show_sensors"] = False
 
-    if picks is None or not _is_numeric(picks):
-        picks = _picks_to_idx(epochs.info, picks)
-        if group_by is None:
-            logger.info("No picks and no groupby, showing the first five "
-                        "channels ...")
-            picks = picks[:5]  # take 5 picks to prevent spawning many figs
-    else:
-        picks = _picks_to_idx(epochs.info, picks)
+    picks = [picks] if isinstance(picks, str) or _is_numeric(picks) else picks
+    too_many_picks = (picks is None or
```

Comment:
make it `many_picks` and it's only 1 line right?

Reply:
nope, not short enough.